survey-runner-queue/terraform.tfvars.example

```diff
@@ -19,4 +19,7 @@ application_cidrs=["10.30.20.48/28"]
 sdx_cidrs=["10.99.0.1/32"]
 rsyslogd_server_ip="10.99.0.2"
 logserver_cidr="10.99.0.3/32"
-audit_cidr="10.99.0.4/32"
\ No newline at end of file
+audit_cidr="10.99.0.4/32"
+
+# Should be true in developer environment to avoid build up of volumes
+delete_on_termination="false"
```

Comment:
Will set to true. Apparently it should be a string according to https://www.terraform.io/docs/configuration/variables.html#booleans

> 
> Although it appears Terraform supports boolean types, they are instead silently converted to string types. The implications of this are subtle and should be completely understood if you plan on using boolean values.
> 
> It is instead recommended you avoid using boolean values for now and use explicit strings. A future version of Terraform will properly support booleans and using the current behavior could result in backwards-incompatibilities in the future.